package-lock.json

```diff
@@ -11786,7 +11786,7 @@
     },
     "medium-zoom": {
       "version": "0.4.0",
-      "resolved": "http://registry.npmjs.org/medium-zoom/-/medium-zoom-0.4.0.tgz",
+      "resolved": "https://registry.npmjs.org/medium-zoom/-/medium-zoom-0.4.0.tgz",
```

Comment:
Hmmmm... why did this change? I'm assuming it was npm, and not you - not sure what the difference is for https vs http in the tar. I suppose that there is a bit more security in the https because the certificate helps prevent spoofing?

Reply:
Yes, it was npm, maybe it was something related to my local environment?
I'm using Node v10.15.3 and npm v6.11.2 on macOS Mojave 10.14.6
I can't find info about this difference, but I don't know if this should be a point of concern for the lib